doc/source/data/examples/ocr_example.ipynb

```diff
@@ -0,0 +1,475 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "9c93fcdf",
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# flake8: noqa\n",
+    "import warnings\n",
+    "import os\n",
+    "\n",
+    "# Suppress noisy requests warnings.\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "os.environ[\"PYTHONWARNINGS\"] = \"ignore\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7854811",
+   "metadata": {},
+   "source": [
+    "# Extracting Information from Documents using Ray Datasets\n",
+    "\n",
+    "In this example, we will show you how to run optical character recognition (OCR) on a set of documents and analyze the resulting text with the natural language processing library spaCy. To make it more interesting, we will run the analysis on the [LightShot](https://www.kaggle.com/datasets/datasnaek/lightshot) dataset. It is a large publicly available OCR dataset with a wide variety of different documents, all of them screen shots of various forms. It is easy to replace that dataset with your own data and adapt the example to your own use cases!\n",
```

Comment:
> run optical character recognition (OCR) on a set of documents and analyze the resulting text with the natural language processing library spaCy. To make it more interesting, we will run the analysis on the dataset

I think this is missing a key component from the user perspective, which is to provide the answer to the question -- "Why do I need Ray for this example?"